packages/bedrock/src/_base.css

```diff
@@ -34,19 +34,38 @@ html {
 *::before,
 *::after { box-sizing: inherit; }
 
-a { color: var(--zd-anchor-color); }
+a {
+  transition:
+    outline-color .1s ease-in-out,
+    color .25s ease-in-out;
+  outline: 2px solid transparent;
+  outline-offset: 1px;
+  border-radius: var(--zd-spacing-xxs);
+  color: var(--zd-anchor-color);
+}
 
 a,
 ins,
 u { text-decoration: none; }
 
-a:hover,
-a:focus {
+a:hover {
   text-decoration: underline;
   color: var(--zd-anchor-color);
 }
 
-a:focus { outline: none; }
+a:hover,
+a:active {
+  color: var(--zd-color-blue-800);
```

Comment:
This does not match `react-components`. https://github.com/zendeskgarden/react-components/blob/e1a2f869c466c4917d4668bc24e2acf42773f178/packages/buttons/src/styled/StyledButton.ts#L67-L70 Hover is blue-700; active is blue-800.

Reply:
That should be corrected now!